subworkflows/local/profiling.nf

```diff
@@ -65,10 +65,14 @@ workflow PROFILING {
         ch_input_for_malt =  ch_input_for_profiling.malt
                                 .filter { it[0]['instrument_platform'] == 'ILLUMINA' }
                                 .map {
-                                    it ->
-                                        def temp_meta =  [ id: it[2]['db_name']]  + it[2]
-                                        def db = it[3]
-                                        [ temp_meta, it[1], db ]
+                                    meta, reads, db_meta, db ->
```

Comment:
Not sure I understand exactly this chunk of code but maybe someone else might review this as well?

Reply:
I'll add somecomments :)

Reply:
Thanks for the comments.